render/fx_renderer/gles2/shaders/gradient.frag

```diff
@@ -0,0 +1,47 @@
+#ifdef GL_FRAGMENT_PRECISION_HIGH
+precision highp float;
+#else
+precision mediump float;
+#endif
+
+vec4 gradient(vec4 colors[LEN], int count, vec2 size, vec2 grad_box, vec2 origin, float degree, bool linear, bool blend){
+	float step;
+
+	vec2 normal = (gl_FragCoord.xy - grad_box)/size;
+	vec2 uv = normal - origin;
+
+	float rad = radians(degree);
+
+	if(linear){
```

Comment:
Nit: please match the bracket spacing throughout the shaders here with the existing format we have in our other shaders